zmapsdk/api.py

```
@@ -8,48 +1,25 @@
-    subnets: List[str]
-    output_file: Optional[str] = None
-
-class StandardBlocklistRequest(BaseModel):
-    output_file: Optional[str] = None
-
-class FileResponse(BaseModel):
-    file_path: str
-    message: str
+import uvicorn
+from fastapi import FastAPI, HTTPException
+from fastapi.responses import FileResponse as StarletteFileResponse
```

Ruff (0.8.2)

`fastapi.responses.FileResponse` imported but unused

Remove unused import: `fastapi.responses.FileResponse`

(F401)